src/algebra/star/self_adjoint.lean

```diff
@@ -270,7 +271,12 @@ instance : has_rat_cast (self_adjoint R) :=
 rfl
 
 instance has_qsmul : has_smul ℚ (self_adjoint R) :=
-⟨λ a x, ⟨a • x, by rw rat.smul_def; exact (rat_cast_mem a).mul x.prop⟩⟩
+⟨λ a x, ⟨a • x, by rw rat.smul_def;
+  begin
+    apply is_self_adjoint.mul,
+    exact rat_cast_mem a,
+    exact x.prop,
+  end ⟩ ⟩
```

Comment:
Ideally we would just prove this with `star_rat_smul`, but that's not imported yet here.

Reply:
And `import algebra.star.module` would be cyclic.